utilities/tree-helpers.metta

```diff
@@ -137,24 +137,22 @@
 
 ;; Given list of trees, intersections will compute the intersection of between their guard sets.
 ;; It has similar behaviour like setIntersection but for more than two trees.
+
 (: intersections (-> (List Tree) (List Tree)))
-(= (intersections $children)
-  (case (length $children) 
-    (
-      (0 Nil)
-      (1 (getGuardSet (head $children)))
-      ($else 
-        (let* 
-          (
-            ($firstChildGuardSet (getGuardSet (head $children)))
-            ($remainingIntersections (intersections (tail $children)))
+(= (intersections Nil) Nil)
+
```

Comment:
Remove Unnecessary spaces between a function definition to be consistent with our code base.